src/index.test.js

```diff
@@ -24,68 +24,87 @@ jest.useFakeTimers();
 
 it("test the form", ()=>{
   const form = mount(<RegistrationForm />);
-
   //检测初始状态
   expect(form.state().permittedname).toBe(false);
   expect(form.state().permittedpsw).toBe(false);
-  expect(form.state().confirmpsw).toBe(false);
+  expect(form.state().confirmedpsw).toBe(false);
   expect(form.state().permittedphone).toBe(false);
   expect(form.state().submitted).toBe(false);
-
   //得到相应input
   const nameinput = form.find({title : "User name:"}).find("input");
   const pswinput = form.find({title : "Password"}).find("input");
   const cpswinput = form.find({title : "Confirm Password:"}).find("input");
   const phoneinput = form.find({title : "Phone Number:"}).find("input");
   const submitbtn = form.find(".submitBtn");
-
-
   //simulate onchange  simulate 可以同时模拟运行多个事件。但由于我的timer都是挂在form组件的this.timer上
   //所以设置另一个时会将之前的抵消
 
+  ////invalid input
```

Comment:
better modify the comments. the comments for test success status and failure status are the same.